Modules/_winapi.c

```diff
@@ -2166,6 +2167,32 @@ _winapi_CopyFile2_impl(PyObject *module, LPCWSTR existing_file_name,
 }
 
 
+/*[clinic input]
+_winapi._sysconfig_vars
+
+Returns a dictionary containing variables intended to be exposed by sysconfig.
+[clinic start generated code]*/
+
+static PyObject *
+_winapi__sysconfig_vars_impl(PyObject *module)
```

Comment:
I don't like this being here, I'd rather it be in the sys module. It's not a Windows API

Reply:
What do you think about putting it in a `_sysconfig` C module?

Reply:
I think we're probably past due having one of those. We already smuggle constants into `getbuildinfo.c` (git info) and `sysmodule.c` (VPATH) on Windows, so may as well put them all into a `_sysconfig.c` that is importable.

Make it a built-in module though, not its own .pyd. `PC/config.c` lists the modules that are directly linked in, so it'll look like one of those.

Reply:
You can see the smuggling in `PCbuild/pythoncore.vcxproj` (search for `GITVERSION=` and `VPATH=`). Doesn't necessarily have to be done the same way, but copying it is going to be the easiest way to ensure incremental builds keep working normally.

Reply:
I've moved the computation to a `_sysconfig` C module. Thanks for the pointer to `PC/config.c` -- that made it easier.  I don't think we need to smuggle data from `PCbuild/pythoncore.vcxproj` like we do for `GITVERSION` and `VPATH` here.